x-pack/solutions/chat/plugins/workchat-app/public/application/components/integrations/listing/integration_list_view.tsx

```diff
@@ -6,63 +6,59 @@
  */
 
 import React from 'react';
-import { EuiBasicTable, EuiBasicTableColumn, EuiButton } from '@elastic/eui';
+import { EuiButton } from '@elastic/eui';
 import { KibanaPageTemplate } from '@kbn/shared-ux-page-kibana-template';
-import { IntegrationType } from '@kbn/wci-common';
-import type { Integration } from '../../../../../common/integrations';
+import { i18n } from '@kbn/i18n';
 import { useNavigation } from '../../../hooks/use_navigation';
 import { appPaths } from '../../../app_paths';
-import { integrationTypeToLabel } from '../utils';
+import { integrationLabels } from '../i18n';
 
-interface IntegrationListViewProps {
-  integrations: Integration[];
-}
-
-export const IntegrationListView: React.FC<IntegrationListViewProps> = ({ integrations }) => {
+export const IntegrationListView: React.FC<{ tab?: string }> = ({ tab }) => {
   const { navigateToWorkchatUrl } = useNavigation();
-  const columns: Array<EuiBasicTableColumn<Integration>> = [
-    { field: 'name', name: 'Name' },
+
+  const renderTabs = () => [
```

Comment:
NIT: Why `renderTabs: () => Tab[]` and then invoking it in the same render via `tabs={renderTabs()}`?

Can't you just do `const tabs = useMemo(() => {....})` and then `tabs={tabs}`

Either way you need memo-ing. (useMemo with my proposal, useCallback otherwise)